ops/testing.py

```diff
@@ -1509,27 +1509,75 @@ def _get_backend_calls(self, reset: bool = True) -> List[Tuple[Any, ...]]:
             self._backend._calls.clear()
         return calls
 
-    def add_model_secret(self, owner: AppUnitOrName, content: Dict[str, str]) -> str:
-        """Add a secret owned by the remote application or unit specified.
-
-        This is named :code:`add_model_secret` instead of :code:`add_secret`
-        to avoid confusion with the :meth:`ops.Application.add_secret`
-        and :meth:`ops.Unit.add_secret` methods used by secret owner
-        charms.
```

Comment:
Do we want to keep this comment (with the method name updated) in `add_charm_secret`? I think it's ok to remove it from the docstring, but I think it would be good to keep as a code comment?